includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -146,7 +146,13 @@ private static function set_payment_method_configuration_cache( $configuration )
 	 * @return object|null
 	 */
 	private static function get_payment_method_configuration_from_stripe() {
-		$result         = WC_Stripe_API::get_instance()->get_payment_method_configurations();
+		$result = WC_Stripe_API::get_instance()->get_payment_method_configurations();
+
+		// If the result does not have a data property, it means we received an error from the API.
+		if ( ! isset( $result->data ) ) {
+			return null;
+		}
```

Comment:
AFAICT, `WC_Stripe_API::retrieve()` can also return `null`, so we should likely check for that case too.
```suggestion
		// If the result does not have a data property, it means we received an error from the API.
		if ( ! is_object( $result ) ||  ! isset( $result->data ) ) {
			return null;
		}
```

Reply:
(took the freedom to fix your suggestion, `isobejct` > `is_object`)